liquidprompt

```diff
@@ -213,6 +213,7 @@ __lp_source_config() {
     LP_ENABLE_RUNTIME=${LP_ENABLE_RUNTIME:-1}
     LP_ENABLE_RUNTIME_BELL=${LP_ENABLE_RUNTIME_BELL:-0}
     LP_ENABLE_VIRTUALENV=${LP_ENABLE_VIRTUALENV:-1}
+    LP_ENABLE_CONTAINER=${LP_ENABLE_CONTAINER:-1}
```

Comment:
The docs say this is disabled by default, but that is not the case here. I think it should be disabled by default.

Reply:
fixed